test/memspace_numa.cpp

```diff
@@ -32,12 +35,17 @@ struct numa_nodes_test : ::umf_test::test {
     }
 
     std::vector<size_t> nodeIds;
+    int skip_this_test;
 };
 
 struct numa_memspace_test : ::numa_nodes_test {
     void SetUp() override {
         ::numa_nodes_test::SetUp();
 
+        if (skip_this_test) {
```

Comment:
could we just create a helper function that checks condition from SetUp() and call it here?

Reply:
All other modified test classes call `::numa_nodes_test::SetUp()` anyway, so there should be only one check in  `::numa_nodes_test::SetUp()`. All `skuip_this_test` flags can be removed.

Reply:
@PatKamin ok originally we called GTEST_SKIP() only in SetUp() and this didn't work - see the description of the PR